doc/Caliptra.md

```diff
@@ -101,7 +101,7 @@ Per [Reference 1](#ref-1), RoT subsystems are required to fulfill three principl
 
 These RoT services can be hosted by a complex RoT as a whole or these services can be spread across one or more components within a platform. This determination has a basis in physical risk. Physical adversaries with reasonable skill can bypass a discrete RoT’s detection capabilities, for example, with SPI interposers.
 
-However, an RoT embedded within a SoC or ASIC represents a much higher detection bar for a physical adversary to defeat. For this reason, Caliptra shall deliver the **Detection** (or Measurement) capability.
+However, an RoT embedded within a SoC or ASIC represents a much higher detection bar for a physical adversary to defeat. For this reason, Caliptra shall deliver the **Detection** capability for itself while providing **Measurement** and **Identity** services for the rest of the SoC. The **Measurement** and **Identity** services provided can be utilized by the SoC to create **Detection** capability for the measured firmware and configuration data.
```

Comment:
`The **Measurement** and **Identity** services provided can be utilized`

maybe reword for simplicity and directness, as so:

`The **Measurement** and **Identity** services that Caliptra provides can be used`

Reply:
done